dom.bs

```diff
@@ -3437,6 +3450,14 @@ method, when invoked, must run these steps:
  <var>options</var>'s {{MutationObserverInit/characterData}} is omitted, then set
  <var>options</var>'s {{MutationObserverInit/characterData}} to true.
 
+ <li><p>If <var>options</var>'s {{MutationObserverInit/elementHasAttributeFilter}} is present and
+ <var>options</var>'s {{MutationObserverInit/childList}} is omitted, then set
```

Comment:
Remind me why we decided that `childList` should be set to true when either of these options are enabled. I think it's ok to just throw typeError, no?

Reply:
It implicitly defaults to checking the immediate node's `childList`. Otherwise it's a bit wonky to have to set  `elementHasAttributeNameFilter` and set `childList` to true every time you want to use `elementHasAttributeNameFilter`